charts/project-origin-wallet/templates/deployment-wallet.yaml

```diff
@@ -104,6 +104,8 @@ spec:
           {{ end }}
             - name: jwt__audience
               value: {{ .Values.config.jwt.audience }}
+            - name: jwt__authority
```

Comment:
Is this what others call an issuer?

Reply:
Authority is the wording used when reference something like a openid-configuration, in that spec, as far as i can see the authority and issuer is the same, but that is not always guaranteed. 

You can configure multiple allowed issuers, but if you use an authority reference, then only a single is possible.